core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -103,14 +107,23 @@ object BattleDamage {
                     .isCityState() && civInfo.hasUnique("+30% Strength when fighting City-State units and cities")
             )
                 modifiers["vs [City-States]"] = 30
+            
+            if (combatant.getTile().isFriendlyTerritory(combatant.getCivInfo())) {
+                val friendlyTerritoryModifier = civInfo.getMatchingUniques("+[]% Strength while fighting in friendly territory")
+                    .fold(0) { sum, it -> sum + it.params[0].toInt() }
```

Comment:
Isn't that exactly the same as `.sumBy { it.params[0].toInt() }`? And if it's not I want to learn the difference!

Reply:
It was probably me trying to do something simple in a complicated way :)